gym_sts/rl/mvp.py

```diff
@@ -165,10 +185,12 @@ def main(_):
     tune_config = TUNE.value
     sync_config = tune.SyncConfig(**tune_config["sync_config"])
     checkpoint_config = config.CheckpointConfig(**tune_config["checkpoint_config"])
+    failure_config = config.FailureConfig(**tune_config["failure_config"])
```

Comment:
We're doing a lot of direct key-based access of values in these dict flags. The fancyflags docs consider this an antipattern, see https://github.com/deepmind/fancyflags#tips. That said, I don't think it's necessary to address it in this PR. Just flagging for posterity.

Reply:
Left a note in the code